staker/manage_pool_tiers.gno

```diff
@@ -177,13 +169,17 @@ func setPoolTier(pool string, tier uint64) {
 
 	// panic if pool does not exist
 	if !(pl.DoesPoolPathExist(pool)) {
-		panic(ufmt.Sprintf("[STAKER] manage_pool_tiers.gno__SetPoolTier() || pool(%s) does not exist", pool))
+		panic(addDetailToError(errDataNotFound,
+			ufmt.Sprintf("manage_pool_tiers.gno__SetPoolTier() || pool(%s) does not exist", pool),
+		))
 	}
 
 	// panic if pool exists in poolTiers
 	_, exist := poolTiers[pool]
 	if exist {
-		panic(ufmt.Sprintf("[STAKER] manage_pool_tiers.gno__SetPoolTier() || pool(%s) already exists in poolTiers", pool))
+		panic(addDetailToError(errInvalidPoolPath,
```

Comment:
Is this not an `errDataNotFound` error?

Reply:
function is `setPoolTier` which adds new pool and tier.
Therefore it can not add pre-existing pool.